Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
```

Comment:
To understand you. In my Extension class

```
// in initialize() method
$this->app->before(array($this, 'before'));

public function before()
{
    $this->addCss('assets/extension.css');
    $this->addJavascript('assets/start.js', true);
}
```

And then the same for ExampleController class with different filenames to include this files only on the routes served by the controller?